### DISCUSSION TITLE
Add options for speed up algorithms

### DISCUSSION THREAD

**Discussion by DeflateAwning** [PaulWoitaschek/Voice#1467] [Ideas]
My understanding is that there are multiple algorithms to speed up the playback speed of a piece of audio. Audacity, for example, has both a quick mode, and a very slow "high quality" mode.

I'm assuming that Voice currently uses a quick-ish algorithm, which results in the audio becoming a touch distorted and robot-sounding. I'm wondering if there's a way to implement the option to select a more power-intensive/higher quality playback speed change algorithm. 

### Resources:
* [good Medium article that explains the idea of "Time Stretching Algorithms"](https://olehc.medium.com/time-stretching-algorithms-and-audio-warping-techniques-used-in-modern-music-production-63f4d33804d2)
* [examples of algorithms](https://www.pitchtech.ch/Time-Stretching/index.html)
* tons of discussion on Stack Overflow, although a lot of it is centered in the user domain (using audio editing softwares)

**Comment by PaulWoitaschek** [comment]
That's more a topic for https://github.com/google/ExoPlayer as Voice does not implement the playback logic itself.

**Comment by PaulWoitaschek** [comment]
Turns out this is already supported by exoplayer by some degree. Converting to an Issue 👍 
https://github.com/google/ExoPlayer/issues/6575

**Reply by PaulWoitaschek** [reply]
https://github.com/PaulWoitaschek/Voice/issues/1469
